packages/components/src/components/marketplace-list.tsx

```diff
@@ -158,3 +132,112 @@ export const MarketplaceList = ({
     </section>
   );
 };
+
+export interface MarketplaceListItemProps extends React.ComponentPropsWithoutRef<'a'> {
+  item: IMarketplaceItemProps;
+}
+
+export function MarketplaceListItem({ item, ...rest }: MarketplaceListItemProps) {
+  return (
+    <Anchor
+      {...item.link}
+      {...rest}
+      className={cn(
+        'hive-focus @container @lg:gap-6 flex gap-4 rounded-2xl border border-transparent bg-neutral-50 p-6 hover:border-neutral-200/50 hover:bg-neutral-100 dark:bg-neutral-800 dark:hover:border-neutral-700 dark:hover:bg-neutral-700/50 [.green_&]:bg-green-900 [.green_&]:hover:border-green-700/50 [.green_&]:hover:bg-green-800/75',
+        item.link.className,
+      )}
+    >
+      <div
+        className={cn(
+          '@lg:size-16 @2xl:size-[92px] size-16 shrink-0 rounded-lg bg-[--bg] [.green_&]:[background:linear-gradient(135deg,_#68A8B6_0%,_#3B736A_100%)]',
+        )}
+      >
+        <Image
+          {...item.image}
+          placeholder={item.image.placeholder || 'empty'}
+          width="92"
+          height="92"
+          className="aspect-square rounded-lg object-contain"
+        />
+      </div>
+      <div className="flex flex-col">
+        <h3 className="@lg:text-2xl m-0 line-clamp-2 font-medium text-[--fg]">{item.title}</h3>
+        <div className="@lg:text-base mb-2 line-clamp-3 text-sm text-[--fg-80]">
+          {item.description}
+        </div>
+        {item.tags && item.tags.length > 0 && (
+          <TagsContainer className="mt-auto">
+            {item.tags.map(tagName => (
+              <Tag key={tagName}>{tagName}</Tag>
+            ))}
+          </TagsContainer>
+        )}
+        <div className="@lg:text-sm flex flex-wrap gap-x-4 text-xs text-[--fg-80]">
+          <span>
+            Updated <time dateTime={item.update}>{formatDate(item.update)}</time>
+          </span>
+          {item.weeklyNPMDownloads && (
+            <span className="@sm:block hidden">
```

Comment:
is it valid?
```suggestion
            <span className="max-@sm:hidden">
```
By the way, what is the point of these new `@sm` and other classes? maybe we need to update just our tailwind config to use new `screens` values?

<img width="181" alt="image" src="https://github.com/user-attachments/assets/6bccf92e-2131-4605-be1f-ae6fc6079372">

Reply:
Oh, but these are relative to `@container`. So the items get bigger if there is less columns in the grid, and then they get smaller again.